lumibot/strategies/strategy.py

```diff
@@ -366,12 +364,10 @@ def log_message(self, message, color=None, broadcast=False):
         >>> self.log_message('Sending a buy order')
         """
         if color is not None:
-            colored_message = f"{self._log_strat_name()}: {message}"
             colored_message = colored(message, color)
-            logging.info(colored_message)
+            self.logger.info(colored_message)
```

Comment:
I agree with the korbit feedback on this one.  It is bad practice to use something that isn't created in your own __init__() function.  It isn't clear in this case where self.logger comes from or how it is setup. This is causing spaghetti code to be introduced and can cause a false coupling between this low level class and child strategy classes that inherit from it.  For instance, you cannot safely use self.logger in this classes __init__() function itself since it is likely that a super().__init__() call to invoke this function will be called before the logger is setup.  It would be safer to setup the logger here.